cmake/windows/helpers.cmake

```diff
@@ -207,13 +219,13 @@ function(_target_install_obs target)
 
     cmake_path(RELATIVE_PATH CMAKE_CURRENT_SOURCE_DIR BASE_DIRECTORY "${OBS_SOURCE_DIR}" OUTPUT_VARIABLE project_path)
 
-    set(32bit_project_path "${OBS_SOURCE_DIR}/build_x64/${project_path}")
-    set(target_file "${32bit_project_path}/$<CONFIG>/${target}64.${suffix}")
-    set(target_pdb_file "${32bit_project_path}/$<CONFIG>/${target}64.pdb")
+    set(64bit_project_path "${OBS_SOURCE_DIR}/build_x64/${project_path}")
+    set(target_file "${64bit_project_path}/$<CONFIG>/${target}64.${suffix}")
+    set(target_pdb_file "${64bit_project_path}/$<CONFIG>/${target}64.pdb")
     set(comment "Copy ${target} (x64) to destination")
 
     install(
-      FILES "${32bit_project_path}/$<CONFIG>/${target}64.${suffix}"
+      FILES "${64bit_project_path}/$<CONFIG>/${target}64.${suffix}"
```

Comment:
I thought this was for copying 32-bit build files (such as virtual camera and game capture) to the 64-bit target build directory. I'll have to double check what this is doing.

Reply:
That's what the branch above that does (`if(_TIO_x86)`), this is the x64 branch for ARM64 builds (`elseif(_TIO_x64)`).

Reply:
Ah, I think I see. This looks like an old copy/paste error that we just didn't notice before?